compose/config/config.py

```diff
@@ -213,6 +214,9 @@ def find(base_dir, filenames):
             os.getcwd(),
             [ConfigFile(None, yaml.safe_load(sys.stdin))])
 
+    if filename.startswith('./http'):
```

Comment:
How about protocols testing for urllib?
I think we could support usual protocols.